src/examples/todo-app/build.md

```diff
@@ -412,7 +416,7 @@ The Zilla engine configuration defines a flow of named `bindings` representing e
 
 Let's configure `Zilla` for the Tasks API to interact with the `Todo` Kafka Streams service via Kafka topics.
 
-You will add the following bindings to support the Tasks API as shown `zilla.yaml` below. To understand each binding type in more detail please visit [Zilla Runtime Configuration](https://docs.aklivity.io/zilla/reference/zilla.json).
+You will add the following bindings to support the Tasks API as shown `zilla.yaml` below. To understand each binding type in more detail please visit [Zilla Runtime Configuration](/reference/zilla.yaml/README.md).
```

Comment:
This link breaks in markdown because it is absolute, should be relative - `../../reference/zilla.yaml/README.md`.

Please use relative links to markdown files from markdown so there is no assumption about any particular path structure in the deployed site. The markdown content links should be relative and therefore relocatable.

Reply:
Down below, I'm seeing explicit changes from relative to absolute, causing more markdown link breakage, is there some forcing reason or can we use relative links as suggested?

Reply:
When these links get parsed, they are converted into a longer reference URL down to the root with the path parameters set on the build. I found the only two forms that would produce valid URLs are:
```
[/reference/zilla.yaml/README.md](/reference/zilla.yaml/README.md)

[../../reference/zilla.yaml/](../../reference/zilla.yaml/)
```
The path with reference to the full filename is preferable since this file can be at any depth away from the destination and only a change to the destination would break it. In the event of moving the destination file and lookup should be made for any reference to the old location, but the link checker will catch old links if they don't get updated.

Reply:
Updated links to use dot absolute path `[../../reference/zilla.yaml/](../../reference/zilla.yaml/)`